src/azure-cli/azure/cli/command_modules/vm/_image_builder.py

```diff
@@ -423,7 +425,8 @@ def create_image_template(  # pylint: disable=too-many-locals, too-many-branches
     if source_dict['type'] == _SourceType.PLATFORM_IMAGE:
         template_source = ImageTemplatePlatformImageSource(**source_dict)
     elif source_dict['type'] == _SourceType.ISO_URI:
```

Comment:
what's __SourceType defined, should 'ISO_URI' be removed from it.

Reply:
I don't know when it will be supported of disabled in the future. So I prefer keeping it.

Reply:
please check latest swagger. If i'm correct ISO source is removed from 2020-02-14 api-version and not supported anymore in 2019-05-01-preview api-version.

Reply:
I just reserve the type. But CLI will raise an exception and let user know ISO is not supported.